doc/deps.md

```diff
@@ -91,12 +92,63 @@ by other processes which could, in extreme cases, make the system very hard or
 impossible to recover. **Please do NOT add a `sudo` in front of any `make` or
 other commands below, unless *explicitly* specified!**
 
+### ODB
+ODB is an Object Relational Mapping tool, that is required by CodeCompass.
+We are using ODB 2.5 which is not part of the official Ubuntu repositories, so
+it needs to be built manually.
+
+The ODB installation uses the build2 build system.
+(Build2 is not needed for CodeCompass so you may delete it right after the installation of ODB.)
+```bash
+wget https://download.build2.org/0.12.0/build2-install-0.12.0.sh
+sh build2-install-0.12.0.sh --yes --trust yes "<build2_install_dir>"
+```
+
+Now, utilizing the *build2* toolchain, we can build the *odb* library. In the script below, we assume that ODB is built in the `<odb_build_dir>` directory and installed in the `<odb_install_dir>` directory. A good default install directory could be `/usr/local/`.
```

Comment:
Actually it's very easy to clean up what ODB-2.5 installs even manually. In single-user developer environment self-compiled tools can go to `/usr/local`, because that is the folder for it by definition. A production system is different.
But we can remove that line.